public/app/main.js

```diff
@@ -38,9 +38,12 @@
     // Boot the application
     require([ 'jquery' ], function () {
         require([ 'bootstrap' ], function () {
-            // TODO Environments other than `development` via build process.
-            require([ 'config/env/development', 'util/router' ], function (environment, router) {
-                environment();
+            var environment = 'development';
+            if (environment !== 'production') {
+                console.log('Running in "' + environment + '" environment');
```

Comment:
In IE lte 9 console is undefined unless developer tools is enabled, but this is in development so I think this is OK.